src/commands/worked-hours.php

```diff
@@ -27,7 +27,7 @@ function workedHours($context, $command) {
       $movementId,
       $timestamp
     ])[0]);
-
+    updateTimesheets($movementId);
```

Comment:
It's probably cleaner to create a separate command `update-timesheets` instead of triggering this from the `worked-hours` command.

Is this function idempotent by the way?

Reply:
We can do that too!
The idea behind was to automate the process.